actionmailer/lib/action_mailer/base.rb

```diff
@@ -900,15 +913,19 @@ def collect_responses(headers)
           yield(collector)
           collector.responses
         elsif headers[:body]
-          [{
-            body: headers.delete(:body),
-            content_type: self.class.default[:content_type] || "text/plain"
-          }]
+          collect_responses_from_text(headers)
         else
           collect_responses_from_templates(headers)
         end
       end
 
+      def collect_responses_from_text(headers)
+        [{
+          body: headers.delete(:body),
+          content_type: headers[:content_type] || self.class.default[:content_type] || "text/plain"
```

Comment:
From what I try and debug, they are different value, because they comes from `mail` method and class default